backend/operations_scanner.go

```diff
@@ -567,3 +575,49 @@ func convertClusterStatus(clusterStatus *arohcpv1alpha1.ClusterStatus, current a
 
 	return opStatus, opError, err
 }
+
+func (s *OperationsScanner) convertInflightChecks(ctx context.Context, internalId ocm.InternalID) (*arm.CloudErrorBody, error) {
+	inflightChecks, err := s.clusterService.GetClusterInflightChecks(ctx, internalId)
+	if err != nil {
+		return &arm.CloudErrorBody{}, err
+	}
+
+	var cloudErrors []arm.CloudErrorBody
+	for _, inflightCheck := range inflightChecks.Items() {
+		if inflightCheck.State() == "failed" {
+			cloudErrors = append(cloudErrors, arm.CloudErrorBody{
+				// TODO: set a custom inflight error?
+				Code:    arm.CloudErrorCodeInternalServerError,
```

Comment:
I'm still tempted to use `inflightCheck.Name()` for the code, at least initially.

Reply:
I'm on the fence here, I think there is no harm in specifying an error code per inflight. 
I need to go through the inflight service code to understand if there is a distinction in CS between a failure due to the cluster spec versus an internal error in the inflight service.

@miguelsorianod any second thoughts on the error code per inflight?

Reply:
Checking the inflight service in CS code, there isn't a distinction between a bad request and an internal server error. In both cases, the inflight will fail with the error details.
Maybe that's an improvement we want to consider before making this distinction in the RP?